.github/workflows/test-fuzz.yml

```
@@ -0,0 +1,78 @@
+name: Test Fuzz on Comment
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+
+env:
+  IMG_REGISTRY: ghcr.io
+  IMG_ORG: kubeflow
+  IMG_REPO: model-registry/server
+  IMG_VERSION: cicd
+  PUSH_IMAGE: false
+
+jobs:
+  check-comment:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/test-fuzz')
+    runs-on: ubuntu-latest
+    outputs:
+      should-run: ${{ steps.check.outputs.result }}
+    steps:
+      - name: Check if comment should trigger test-fuzz
+        id: check
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const comment = context.payload.comment.body.trim();
+            const isExactMatch = comment === '/test-fuzz';
+            const isLineMatch = comment.split('\n').some(line => line.trim() === '/test-fuzz');
+            return isExactMatch || isLineMatch;
+
+  test-fuzz:
+    needs: check-comment
+    if: needs.check-comment.outputs.should-run == 'true'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: clients/python
+    steps:
+      - name: Get PR details
+        id: pr
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+            return {
+              sha: pr.data.head.sha,
+              ref: pr.data.head.ref
+            };
+
+      - name: Checkout PR
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ fromJson(steps.pr.outputs.result).sha }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install Poetry
+        run: |
+          pipx install poetry
+
+      - name: Remove AppArmor profile for mysql in KinD on GHA
```

```suggestion
      - name: Remove AppArmor profile for mysql in KinD on GHA # https://github.com/kubeflow/manifests/issues/2507
```

this way we can keep track by cross-linking